File: deploy.js
import { spawn } from 'child_process';
const child = spawn('ssh', ['-o', 'StrictHostKeyChecking=no', 'root@65.21.244.158', 'uptime'], {
  stdio: ['pipe', 'pipe', 'pipe']
});
child.stdout.on('data', (data) => console.log('STDOUT: ' + data));
child.stderr.on('data', (data) => {
    const str = data.toString();
    console.log('STDERR: ' + str);
    if (str.toLowerCase().includes('password')) {
        console.log('Password prompt detected, sending password...');
        child.stdin.write('Cph181ko!!\n');
    }
});
child.on('close', (code) => console.log('Process exited with code: ' + code));

File: deploy.py
import subprocess
import time

print("Starting SSH process...")
p = subprocess.Popen(['ssh', '-o', 'StrictHostKeyChecking=no', 'root@65.21.244.158', 'uptime'], stdin=subprocess.PIPE, stdout=subprocess.PIPE, stderr=subprocess.PIPE, text=True)
try:
    print("Sending password...")
    stdout, stderr = p.communicate(input='Cph181ko!!\n', timeout=20)
    print("STDOUT:", stdout)
    print("STDERR:", stderr)
except Exception as e:
    print("Error:", e)
    p.kill()
